src/compile/data/summary.ts

```diff
@@ -48,14 +47,14 @@ export namespace summary {
 
           // add min/max so we can use their union as unaggregated domain
           const scale = model.scale(channel);
-          if (scale && useUnaggregatedDomain(scale, model, channel)) {
+          if (scale && scale.domain === 'unaggregated') {
```

Comment:
Data is parsed before scales are parsed. However, we initialize scales before we parse them so we will have this property set correctly. This is the main idea of this PR, but I just want to make sure I understand it correctly.